demo/server/src/app.ts

```diff
@@ -1,33 +1,112 @@
 'use strict';
 
 import * as express from 'express';
+import * as session from 'express-session';
 import * as path from 'path';
 import * as logger from 'morgan';
 import * as cookieParser from 'cookie-parser';
 import * as bodyParser from 'body-parser';
 import index from './routes/index';
 import * as favicon from 'serve-favicon';
 import EnvironmentConfig, { Config, CloudAppConfig } from './util/config';
+// import * as passport from 'passport';
+// import {Strategy} from 'passport-local';
+import users, {User} from './userSeed';
+import PassportAuthImpl from '@raincatcher/passport-auth';
 
 const app: express.Express = express();
 const appConfig: Config<CloudAppConfig> = new EnvironmentConfig<CloudAppConfig>();
 const config = appConfig.getConfig();
+// const authService = new PassportAuthImpl<User>();
 
 app.use(logger(config.morganOptions));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
+app.use(session({secret: 'test', resave: false, saveUninitialized: false}));
```

Comment:
[Hint] if there are some strict requirements for top level app that aren't part of the module try to wrap them into separate class (similar to webapi aproach) - this way users will clearly know what elements they need to add and you can write documentation snippents pointing to this rather than maintaining chunks of data in docs.